Source/WTF/wtf/CurrentTime.h

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
Why is this new file needed? Wasn't it moved elsewhere?

Reply:
But this file is under performance tests but building required this file . So i moved it here.
🤔